tests/command/toggleattributecommand.js

```diff
@@ -5,7 +5,7 @@
 
 import Editor from '/ckeditor5/core/editor/editor.js';
 import Document from '/ckeditor5/engine/model/document.js';
-import AttributeCommand from '/ckeditor5/core/command/attributecommand.js';
+import ToggleAttributeCommand from '/ckeditor5/core/command/toggleattributecommand.js';
```

Comment:
And I'm still thinking about `BooleanAttributeCommand` :D. But from the two you proposed, I'm more for `ToggleAttributeCommand`. It's more in line with the names that we always used (in terms of grammar).